### PR TITLE
Feature/apply page

### PR DESCRIPTION
**Pull request by DeveloperKhan**
Apply page!

### CODE REVIEW

File: pages/apply/index.js [hunk 1]
```diff
@@ -1,15 +1,91 @@
...
+  const router = useRouter();
+  const currentRoute = router.pathname;
+  const content = currentRoute === chRoute ? pretendImContentful1 : pretendImContentful2;
   return (
     <main>
       <Header title="Apply">
         <div className={styles.switch_control}>
-          <a>For New Chapters</a>
-          <a>For Non Profits</a>
+          <a className={classNames({ [styles.selected]: currentRoute === chRoute })} href={chRoute}>
```
Comment (ghost) [inline pages/apply/index.js:59]: Use a `<Link>` boi here

File: pages/apply/index.js [hunk 2]
```diff
@@ -1,15 +1,91 @@
...
+          </a>
+          <a className={classNames({ [styles.selected]: currentRoute === npRoute })} href={npRoute}>
+            For Non Profits
+          </a>
         </div>
       </Header>
+      <div style={{ marginTop: 20 }}>
+        <Container className="row spaced aligned wrap">
+          <div className={styles.requirements}>
+            <h2>{"What you'll need"}</h2>
```
Comment (ghost) [inline pages/apply/index.js:70]: Nit: No need for the curly braces here

**Review by ghost — CHANGES_REQUESTED** [review]

File: pages/apply/index.js (hunk 1 above)
Comment (DeveloperKhan) [reply pages/apply/index.js:59]: fixed

File: pages/apply/index.js (hunk 2 above)
Comment (DeveloperKhan) [reply pages/apply/index.js:70]: eslint

**Pull request merged**